### ISSUE TITLE
Police Oversight form: Implement WCAG compliant date picker

### ISSUE THREAD

**Issue by amenity** [cityofaustin/techstack#1446]
![datepicker](https://user-images.githubusercontent.com/1463708/54229570-88c5dc00-44d2-11e9-875d-09ee77bafe78.jpg)

Also, per @toribr - 
- [ ] Date and time format after entry should be human readable format: 
NO: 
![screen shot 2019-03-04 at 12 47 37 pm](https://user-images.githubusercontent.com/3790067/53755413-19217280-3e7c-11e9-9af1-8c3421043635.png)
YES (as seen on review screen)(plain text on what-happened instance, please):
![screen shot 2019-03-04 at 12 49 58 pm](https://user-images.githubusercontent.com/3790067/53755405-145cbe80-3e7c-11e9-8c0d-bf47b203982c.png)

**Comment by amenity** [comment]
![image](https://user-images.githubusercontent.com/1463708/54463886-8a95d680-4742-11e9-87ea-ed048f144bfc.png)

**Comment by oliver2g** [comment]
I'm doing QA on this.

Mobile:
- [ ] iPhone: Safari 12.0
- [ ] Android: Chrome 71.0

Desktop:
- [ ] Windows 10: Chrome 71.0
- [ ] Windows 7: Chrome 71.0; 
  - [ ] Internet Explorer 11
- [ ] MacOS 10.14: Chrome 71.0
- [ ] Windows 10: Edge 17.17; 
  - [ ] Internet Explorer 11

**Comment by easherma** [comment]
LGTM! However, this did surface #1592 and #1593
